validator/main.py

```diff
@@ -55,12 +56,14 @@ def chunking_function(self, chunk: str):
     def __init__(
         self,
         competitors: List[str],
+        use_local: bool = False,
```

Comment:
Move this below on_fail to preserve argument positions or else this could be breaking for existing users.

Also should use_local be optional?